DQMOffline/Trigger/python/HiggsMonitoring_cff.py

```diff
@@ -1,4 +1,141 @@
+
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.HiggsMonitoring_cfi import hltHIGmonitoring
+
+##############################DiLepton cross triggers######################################################
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref = hltHIGmonitoring.clone()
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.nmuons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.nelectrons = cms.uint32(1)
+mu23TrkIsoVVLEle8CaloIdLTrackIdLIsoVL_muref.FolderName = cms.string('HLT/Higgs/DiLepton/HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL_muref/')
```

Comment:
probably a silly commnet
but wouldn't it be easier to have a directory structure like 
HLT/Higgs/DiLepton/HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL/mu leg
and
HLT/Higgs/DiLepton/HLT_Mu23_TrkIsoVVL_Ele8_CaloIdL_TrackIdL_IsoVL/ele leg

in addition, do you really need the `Higgs` sub-directory ?